db/migrate/20150928180532_add_user_id_to_notes.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserIdToNotes < ActiveRecord::Migration
+  def change
+    add_column :notes, :user_id, :integer
```

Comment:
The foreign key should also have an index. Admittedly, the table won't go Big Data anytime soon, but it's good to make a habit out of it.